specification/maps/data-plane/Route/preview/1.0/route.json

```diff
@@ -566,86 +566,90 @@
       "x-ms-client-name": "FilterSectionType",
       "in": "query",
       "description": "Specifies which of the section types is reported in the route response. <br><br>For example if sectionType = pedestrian the sections which are suited for pedestrians only are returned. Multiple types can be used. The default sectionType refers to the travelMode input. By default travelMode is set to car",
```

Comment:
We need to modify the "description" and make is clear it can be specified multiple times in one request, for example, '&sectionType=carTrain&sectionType=pedestrain&sectionType=motorway', using comma separated values will not work.

It has the same usage like `avoid` parameter. See the code and how it describe in description section:
https://github.com/Azure/azure-rest-api-specs/blob/31c94bc93e096644eb9b56af6872ebcfd3043e19/specification/maps/data-plane/Route/preview/1.0/route.json#L296

Reply:
@dubiety done